app/ts/components/simulationExplaining/SimulationSummary.tsx

```diff
@@ -464,12 +464,14 @@ export function TokenLogAnalysisCard({ simTx, renameAddressCallBack }: TokenLogA
 	const [showLogs, setShowLogs] = useState<boolean>(false)
 	const identifiedSwap = identifySwap(simTx)
 	if (simTx === undefined) return <></>
-
+	const hasEthLogs = simTx.transaction.rpcNetwork.httpsRpc === 'https://rpc.dark.florist/birdchalkrenewtip' || simTx.transaction.rpcNetwork.httpsRpc === 'https://rpc.dark.florist/winedancemuffinborrow' //todo, remove this check, all txs should have ETH logs
+	const tokenEventsPlurar = hasEthLogs ? 'token events or ETH transactions' : 'token events'
```

Comment:
this might be a typo for `tokenEventsPlural`. 

*This seem to appear in multiple places so can't do suggestion comment on this one.*